ci/jjb/jobs/macros.yaml

```diff
@@ -56,7 +56,7 @@
             hostname --all-fqdn
             hostname
             # Setting the hostname of the system to enable connectivity
-            hostname="$(hostname --all-fqdn | grep openstacklocal | head -n 1)"
+            hostname="$(hostname --all-fqdn | grep upshift | head -n 1)"
```

Comment:
@rochacbruno I think if you rebase against `production master`, this has already been merged in.